pages/index.tsx

```diff
@@ -91,15 +91,15 @@ const Index: NextPage<IndexProps & PlayerControls> = ({
         navigationItems={navigationItems}
         isCompact={isPlayerLive}
       />
-      {/* {isPlayerLive && ( */}
-      {/*   <Player */}
-      {/*     playing={playing} */}
-      {/*     onPlayPause={onPlayPause} */}
-      {/*     muted={muted} */}
-      {/*     onMute={onMute} */}
-      {/*     showsByDate={showsByDate} */}
-      {/*   /> */}
-      {/* )} */}
+      {/* {isPlayerLive && (
```

Comment:
Soitin ei taida vielä olla näkyvillä?

Reply:
Juu, näin juuri.

Reply:
Voisit pistää tällaisena menemään tuotantoon nyt ja sitten julkaistaan deviin ohjelmakartan kanssa 👍